ush/find_fcst_length.sh

```diff
@@ -10,9 +10,9 @@ array=($1)
 cyc=$2
 fcst_length=$3
 num_fhrs=${#array[@]}
-if [[ $num_fhrs -eq 24 ]]; then
+if (( $num_fhrs == 24 )); then
   icyc=$((10#${cyc}))
-  if [ $icyc -lt $num_fhrs ]; then
+  if (( $icyc < $num_fhrs )); then
```

Comment:
Line 13 and 15,  suggest using `${num_fhrs}` and `${icyc}` instead of `$num_fhrs` or `$icyc` to be consistent with other parts on the variable reference style.

Reply:
Done